### PR TITLE
Update to intellij 2024.2+

### PR DESCRIPTION
**Pull request by Minecraftschurli**

### CODE REVIEW

File: .github/workflows/build.yml
Comment (EnzDev) [inline .github/workflows/build.yml:1]: Hello, there is some small parts that needs changes from what I'm seeing in the [IntelliJ template repository](https://github.com/JetBrains/intellij-platform-plugin-template/blob/main/.github/workflows/build.yml).

`Export Properties` job should not execute `./gradlew listProductsReleases # prepare list of IDEs for Plugin Verifier` anymore.
export properties

[Sources should be fetched with full history before running Qodana](https://github.com/JetBrains/intellij-platform-plugin-template/commit/e6049b22835567f429a45d4dce4881c1728bcd84)
fetch pr

Plugin verifier command has changed
plugin verifier

**Review by EnzDev — CHANGES_REQUESTED** [review]
Thanks for your help ! It seems your merge request needs some small changes for the CI to pass, I'll need it to deploy the plugin 😉

**Comment by EnzDev** [comment]
If you prefer it is also possible for me to add you as a contributor so you can create a branch with your changes so we can collaborate

**Comment by EnzDev** [comment]
I'll merge to next to allow me to do the final CI fixes :) Thanks !

**Comment by Minecraftschurli** [comment]
Oh I was working on the fix for the verify task

**Comment by EnzDev** [comment]
> Oh I was working on the fix for the verify task
Really sorry for that, I wasn't sure the CI was working automatically for you because I had to validate to run them.
Thanks a lot again for your help :)

**Pull request merged**